tests/util/mod.rs

```diff
@@ -137,6 +137,20 @@ impl<'a> TestManager<'a> {
 
         Ok(())
     }
+
+    /// Remove a temporary table from the list of temporary tables.
+    pub fn remove_temporary_table(
+        &mut self,
+        table_name: String,
+    ) -> Result<(), Box<dyn std::error::Error>> {
+        if self.temporary_tables.contains(&table_name) {
```

Comment:
I think this check is unnecessary. If you want to ensure the deletion, we should check the change of the length of `self.temporary_tables`.

Reply:
I felt that determining whether a temporary table has been deleted or not should not be done in this function, so I will remove that processing